vm/ubpf_vm.c

```diff
@@ -298,13 +299,14 @@ ubpf_exec(const struct ubpf_vm* vm, void* mem, size_t mem_len, uint64_t* bpf_ret
 
     stack = calloc(UBPF_STACK_SIZE, 1);
     if (!stack) {
-        return -1;
+        return_value = -1;
+        goto exit;
```

Comment:
I don't see a definition of the `exit` label

Reply:
Typo.